cmd/ops_agent_windows/run_windows.go

```diff
@@ -67,14 +77,55 @@ func (s *service) parseFlags(args []string) error {
 	return fs.Parse(allArgs)
 }
 
+func (s *service) checkForStandaloneAgents(unified *confgenerator.UnifiedConfig) error {
+	mgr, err := mgr.Connect()
+	if err != nil {
+		return fmt.Errorf("failed to connect to service manager: %s", err)
+	}
+	defer mgr.Disconnect()
+	services, err := mgr.ListServices()
+	if err != nil {
+		return fmt.Errorf("failed to list services: %s", err)
+	}
+
+	var errors string
+	if unified.HasLogging() && containsString(services, "StackdriverLogging") {
+		errors += "We detected an existing Windows service for the StackdriverLogging agent, " +
```

Comment:
I realize I'm entering crazy town with this suggestion, but, if only one of the agents conflicts should we still start the other agent?

Reply:
My current thinking is to keep it as is.

This way will force the user to decide between disabling the standalone agent or disabling the config section.  Either action is easy to do.

If we were to start the non-conflicting sub-agent, the user might be confused that 1/2 of the config is being ignored.